state/testing/suite_wallclock.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package testing
+
+import (
+	"github.com/juju/testing"
+	gc "gopkg.in/check.v1"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/juju/cloud"
+	"github.com/juju/juju/environs/config"
+	"github.com/juju/juju/state"
+	coretesting "github.com/juju/juju/testing"
+	"github.com/juju/juju/testing/factory"
+)
+
+var _ = gc.Suite(&StateWithWallclockSuite{})
+
+// StateWithWallclockSuite provides setup and teardown for tests that require a
+// state.State. This should be deprecated in favour of StateSuite, and tests
+// updated to use the testing clock StateSuite provides.
+type StateWithWallclockSuite struct {
```

Comment:
Already merged, but work around this area continues, so I'll update this very soon. Thanks.